### ISSUE TITLE
Video Chat Message

### ISSUE THREAD

**Issue by timothytclee** [badoo/Chatto#343]
First of all, thank you to everyone who has contributed to this great project.

Wondering how easy it would be to add video input and video messages - since there is already LiveCameraCaptureSession.  Has anybody tried this?

Thanks!

**Comment by AntonPalich** [comment]
@timothytclee I didn't try It, but it should be easy, we've added a few input items and dozen of chat messages in Badoo app. Any specific questions?

**Comment by mohammedelsammak** [comment]
@timothytclee Have you managed to make it?

**Comment by timothytclee** [comment]
@mohammedelsammak Yes, I did. I ended up building my own ChatInputBar that calls a standard camera/media picker that supports both photo and video capture.

**Comment by Denismih** [comment]
@timothytclee Could you please share the cod
